pandas/tseries/index.py

```diff
@@ -64,6 +64,7 @@ def f(self):
             if self.tz is not utc:
                 values = self._local_timestamps()
 
+        # boolean accessors -> return array
```

Comment:
In principle, that is indeed cleaner. But, the problem is that I would still have to distinguish here in another way, as the `is_leap_year` is also a boolean one, but has to be processed differently. So not sure if that is then worth it.